packages/grafana-ui/src/components/Select/SelectBase.tsx

```diff
@@ -124,7 +125,7 @@ export function SelectBase<T>({
   menuPlacement = 'auto',
   menuPosition,
   menuShouldPortal = true,
-  noOptionsMessage = 'No options found',
+  noOptionsMessage = t('select.select-base.no-options-found-label', 'No options found'),
```

Comment:
for both these phrases in grafana-ui, lets use the message IDs
 - `grafana-ui.select.no-options-label`
 - `grafana-ui.select.placeholder`

We want to have the grafana-ui translations under their own section in the extracted message files so they're nicely organised ✨

Reply:
Thank you, I made the updates.